src/content/docs/github-steps.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Github Steps
+---
+
+## When starting a new project, visit Github to "fork" code.
+
+
+1. Click “Your Profile” in top right hand corner.
+2. Click “Repositories”.
+3. Click the repo you’d like to fork.
+4. Fork in top right hand corner. Create fork.
+5. Click Code then Copy git url.
+6. Open “Command Prompt”
+7. Type `dir` for a list of folders and press enter.
+8. Type `cd folder the project should go in` then press enter.
+9. Type `git clone https://github.com/vchinnick/eleventy-starter-project.git url`
+10. In VS code, click ‘File’ then ‘Open folder’ then choose the folder your project is in.
+11. At the top, click ‘Terminal’ then ‘New Terminal’
+12. In VS code, create new branch.
+    - `Git checkout -b (name)`
+13. To make sure you’re on the right branch, type `git branch`.
+14. Start adding stuff to the editor. Save when you’re done adding edits.
+15. To see changes, run the project locally [npm start].
+16. If in Node, switch to powershell.
+17. Under Source Control, if it’s listed as Untracked–right click and Stage changes OR in terminal use command below..
+    - `git add .` for all files
+    - `git add file-name` for specific files.
+18. Commit changes to branch
+    - `git commit -am ‘message’`
+19. Push changes to Github
+    - `git push origin head`
+    - OR use Publish button to publish branch-to origin
```

Comment:
```suggestion
## Committing Changes to a Branch

1. Check for any untracked files
    - You can check the status of your changes by running `git status` inside the terminal.
    - You can also check the status of changes within VS Code's Source Control tab, 
      - Files marked with `M` mean they have been "modified" in the branch.
      - Files marked with `U` mean they are "untracked" in the branch.
      - Files marked with `D` mean they have been "deleted" from the branch.
      - Files marked with `A` mean they have been "added" to the branch.
2. If a file is untracked – it will need to be added to the branch as a tracked file.
    - In the terminal you can add files by entering:
      - `git add file-name` for specific files.
      - `git add .` for all files
    - You can add files through VS Code's Source Control tab as well, by right-clicking the file and choosing "Stage Changes"
3. If there are changes you do not want to be committed to the branch, you can revert individual files or all files before committing.
4. Once all changes have been staged, you can commit changes to the branch running:
    - `git commit -am ‘commit message here’`
5. Push changes to GitHub
    - In the terminal, enter `git push origin head` to push the current working branch to the remote origin repository on GitHub
    - You can also push a branch through VS Code's Source Control tab by pressing the "Publish Branch" button whenever it is available.
```